Controllers/FileWriter.cs

```diff
@@ -9,9 +9,10 @@ internal class FileWriter
     {
         public void Export()
         {
-            var saveFileDialog = new SaveFileDialog();
-            var filter = "CSV file (*.csv)|*.csv| All Files (*.*)|*.*";
-            saveFileDialog.Filter = filter;
+            var saveFileDialog = new SaveFileDialog
+            {
+                Filter = @"CSV file (*.csv)|*.csv| All Files (*.*)|*.*"
```

Comment:
Maybe it is better to set Filter to `@"CSV file (*.csv)|*.csv"` since our application accepts only csv format?